include/vcpkg/base/files.h

```diff
@@ -25,6 +25,14 @@ namespace stdfs = std::experimental::filesystem;
 
 namespace vcpkg
 {
+    struct IgnoreErrors
+    {
+        operator std::error_code&() { return ec; }
```

Comment:
No suggestion -- just noting that I haven't seen this interesting sequence before (`operator error_code&` on a temporary).

Reply:
It isn't a thing I've seen before either :)